controllers/topics/index.js

```diff
@@ -238,6 +239,7 @@ const followTopicV2 = async (req, res) => {
       }
 
       await userTopicService.followTopic(detailTokenUser.user_id, topic_id);
+      await _topicAutoMessage(userId, topic_id, detailTokenUser);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `_topicAutoMessage` is called correctly with the appropriate parameters.
```diff
+       await _topicAutoMessage(userId, topic_id, detailTokenUser);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->